db/migrate/20240507200527_update_ivc_champva_forms_indices.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class UpdateIvcChampvaFormsIndices < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :ivc_champva_forms, :form_uuid, algorithm: :concurrently
+    remove_index :ivc_champva_forms, name: 'index_ivc_champva_forms_on_email'
```

Comment:
Can you first do `remove_index` and then in a follow-up PR, `add_index`? That will simplify this a bit more. I don't think you need to use `disable_ddl_transaction!` for the remove_index.

And when you `add_index` would it make sense to include `unique: true` before `algorithm: :concurrently`?

Reply:
@rmtolmach The index is so lookups can happen faster. form_uuid won't be unique only the auto-generated ID will be.